### DISCUSSION TITLE
[Help needed] Launching Write for AppFlowy

### DISCUSSION THREAD

**Discussion by annieappflowy** [AppFlowy-IO/AppFlowy#2116] [General help wanted]
Hi all, we're launching Write for AppFlowy in a few days!  The program is created for writers who want to help other AppFlowy users and builders, get paid, and build credibility. To learn more, here is a working-in-progress program guide: https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/write-for-appflowy

I'm collecting feedback from the community before the official launch. Please comment under this post with your feedback, suggestions, or questions about the program 🙏 🚀 ✍️

**Comment by JRS296** [comment]
Hey @annieappflowy, I'd love to help out with blog writing and general documentation for AppFlowy. I had gone through the link you've provided. Would you like potential contributions including general use case scenarios for AppFlowy?  Such as how it can be used for general productivity? Or would something more technical be required?

**Comment by annieappflowy** [comment]
Hi @JRS296 , thanks so much for offering the help! I'm planning to partner with a few enthusiastic writers to roll out the program. You can be one of them! We are looking for general use case scenarios and more product-oriented how-to guides to help users learn how to use AppFlowy and unleash its power :D That said, we also welcome technical deep dives.

**Reply by JRS296** [reply]
The thing is, I use Notion extensively, and I have created and used many templates; for different workflows. I can definitely try implementing the same on Appflowy, and can help write for it as well. Please let me know how I can get started. If possible, I can try writing out few proposals for the same, and ideas on how Appflowy can be used.

**Reply by JRS296** [reply]
@annieappflowy Can I try getting started by drafting some proposal's for the same? I shall post it on Discord as required, or I could try creating a new issue for the same.

**Reply by annieappflowy** [reply]
Hi @JRS296 , I added a few ideas to [the program guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/write-for-appflowy#how-to-guides). Please take a look and see if you find an interesting topic :D
To get started, you need to follow the [steps](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/write-for-appflowy#how-does-it-work) explained in the guide.

**Reply by JRS296** [reply]
Sure, looking forward to it!

**Comment by AmanNegi** [comment]
I would like to write on the Kanban board exploring its features and possible use cases with examples. Would this be fine?

**Reply by annieappflowy** [reply]
Absolutely! Please follow the program guidelines to submit your proposal.
